OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs

```diff
@@ -22,8 +22,13 @@ namespace OpenRA.Mods.Cnc.Traits
 {
 	class ChronoshiftPowerInfo : SupportPowerInfo
 	{
-		[Desc("Target actor selection radius in cells.")]
-		public readonly int Range = 1;
+		[FieldLoader.Require]
+		[Desc("Size of the footprint of the affected area.")]
+		public readonly CVec Dimensions = CVec.Zero;
+
+		[FieldLoader.Require]
+		[Desc("Actual footprint. Cells marked as x will be affected.")]
+		public readonly string Footprint = string.Empty;
```

Comment:
It's probably not a big deal, but why `string.Empty` instead of `null` for a field with `[FieldLoader.Require]`?

Reply:
We usually default to `null`, so I'd prefer that over `string.Empty`.

Reply:
This was probably copied from `TerrainTunnel` where we do the same.